tensorflow/lite/micro/kernels/svdf_common.cc

```diff
@@ -487,6 +487,10 @@ TfLiteStatus PrepareSvdf(TfLiteContext* context, TfLiteNode* node) {
             &(data->scratch_output_tensor_index));
     TF_LITE_ENSURE_OK(context, scratch_output_status);
   } else {
+    if (input->type == kTfLiteInt16) {
```

Comment:
SVDF does not support int16 activations. I would recommend leaving this file unchanged.

Reply:
could you revert the changes in svdf_common? or if you think this is needed, please let me know why.

Reply:
My apologies, I was changing the wrong file. I will upload a new PR once it has passed the internal review